src/electron/views/root-container/components/root-container.tsx

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import * as React from 'react';
+
+import { TelemetryPermissionDialogDeps } from 'common/components/telemetry-permission-dialog';
+import { ClientStoresHub } from 'common/stores/client-stores-hub';
+import { UserConfigurationStoreData } from 'common/types/store-data/user-configuration-store';
+import { DeviceStoreData } from 'electron/flux/types/device-store-data';
+import { WindowStateStoreData } from 'electron/flux/types/window-state-store-data';
+import {
+    AutomatedChecksView,
+    AutomatedChecksViewDeps,
+    AutomatedChecksViewProps,
+} from 'electron/views/automated-checks/components/automated-checks-view';
+import { DeviceConnectBodyDeps } from 'electron/views/device-connect-view/components/device-connect-body';
+import { DeviceConnectViewContainer } from 'electron/views/device-connect-view/components/device-connect-view-container';
+
+// root container deps should only depend on ResultsViewContainerDeps and DeviceConnectBodyDeps. This will be addresed in a later PR
```

Comment:
why not now?? I think this can be solve by updating `RootContainerProps` to:
```typescript
export type RootContainerProps = AutomatedChecksViewProps & DeviceConnectViewContainerProps;
```

Reply:
updated it, but then removed it as AutomatedChecksViewProps won't be implemented or used in this PR